package.json

```diff
@@ -207,5 +207,8 @@
     "snapshotSerializers": [
       "enzyme-to-json/serializer"
     ]
+  },
+  "resolutions": {
+    "uglify-es": "3.3.4"
```

Comment:
What is this part for?

Reply:
Mentioned it in the comment, but later versions result in a build error

Reply:
Ah didn't catch that. Thanks!